python/python-guestbook/src/backend/skaffold.yaml

```diff
@@ -0,0 +1,22 @@
+# To learn more about the skaffold.yaml schema visit
+# https://skaffold.dev/docs/references/yaml/
+apiVersion: skaffold/v2beta4
+kind: Config
+build:
+  tagPolicy:
+    sha256: {}
```

Comment:
What do you think about getting rid of these tagPolicies and using the default?

Reply:
That sounds like a good idea to me, especially after reading this great line from the [skaffold.yaml](https://skaffold.dev/docs/references/yaml/#build-tagPolicy) documentation about `tagPolicy`:
"A few strategies are provided here, although you most likely won't need to care!"